src/data/beans/Section.ts

```diff
@@ -116,6 +118,14 @@ export default class Section {
           from: new Date(),
           to: new Date(),
         },
+        finalDate:
+          finalDateIndex === -1 || oscar.version < 3
+            ? new Date()
+            : oscar.finalDates[finalDateIndex] ?? new Date(),
```

Comment:
Any objections to using `null` instead of `new Date()` here? I think it makes the error/non-existant case much clearer.

Reply:
Yeah that makes sense. Not sure why I was creating a new Date object in the first place